### ISSUE TITLE
Insufficient video memory，f1=0.0, loss=nan

### ISSUE THREAD

**Issue by 1aozh1q1ng** [walking-shadow/Official_Remote_Sensing_Mamba#38]
Hello, author, I encountered a problem: I used LEVIR-CD for training. Even though I adjusted the batchsize to a small size, the video memory was still not enough, so I cut the image to 256 and set batchsize=8. When training to the 119th epoch, the error of f1=0.0, loss=nan occurred. Please tell me how to solve the above problem.
There is another question. In the ph file, there is an image_size parameter setting, which corresponds to crop_size in utils. That is to say, input 1024 images and the images will be cropped before training. However, as mentioned before, directly inputting 1024 images will cause insufficient video memory, so I cropped the data set to 256 before inputting, but this caused the above problem.

**Comment by keep123ing** [comment]
你好，方便问一下您复现成功了吗，我纯小白一个，好多报错，如果您复现成功可不可以告诉我一下原代码有哪些需要修改的地方吗

**Comment by MasaHiroSaber** [comment]
I encountered the same issue while training LEVIR-CD+. When it reached the 58th epoch, the train F1 score = 0.0, and the train epoch loss= nan.

**Comment by MasaHiroSaber** [comment]
I encountered the same issue while training LEVIR-CD+. When it reached the 25th epoch, the train F1 score = 0.0, and the train epoch loss = nan.

![Image](https://github.com/user-attachments/assets/7ab9eba6-f3cc-4184-a660-f5986f8f18af)

**Comment by Assioncreed** [comment]
> I encountered the same issue while training LEVIR-CD+. When it reached the 25th epoch, the train F1 score = 0.0, and the train epoch loss = nan.
> 
> ![Image](https://github.com/user-attachments/assets/7ab9eba6-f3cc-4184-a660-f5986f8f18af)

I have the same problem. Did you solve it?

**Comment by ismutao** [comment]
可以尝试一下降低一点学习率和衰减权重，提高batchsize的大小，同时改变对应的warm_up

**Comment by BlushMaker** [comment]
> 可以尝试一下降低一点学习率和衰减权重，提高batchsize的大小，同时改变对应的warm_up

那样的话就OutOfMemoryError了
